src/core/create_compilers/RollupCompiler.ts

```diff
@@ -300,6 +333,14 @@ export default class RollupCompiler {
 			}
 		};
 
+		mod.plugins.push(css_hashing);
```

Comment:
line 339 does `mod.plugins || (mod.plugins = [])`. it seems like this should come after that. maybe something like:
```
mod.plugins = mod.plugins || [];
mod.plugins.push(css_hashing);
mod.plugins.push(css_chunks({ injectImports: true }));
```